timesheets/Code.gs

```diff
@@ -12,25 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
  
-// GLOBAL VARS
-var HOURS_START = 4;
-var HOURS_END = 8;
-var HOURLY_PAY_COLUMN = 9;
-var APPROVAL_COLUMN = 11;
-var EMAIL_COLUMN = 3;
+// Global variables representing the index of certain columns.
+var COLUMN_INDEX = {
```

Comment:
I think these numbers represent column numbers (start with 1) vs column indices (start with 0). Change to `COLUMN_NUMBERS` instead.